content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -156,11 +156,15 @@ kubeadm config images pull
 You can pass `--config` to the above commands with a [kubeadm configuration file](#config-file)
 to control the `kubernetesVersion` and `imageRepository` fields.
 
-All default `k8s.gcr.io` images that kubeadm requires support multiple architectures.
+All default `registry.k8s.io` images that kubeadm requires support multiple architectures.
 
 ### Using custom images {#custom-images}
 
-By default, kubeadm pulls images from `k8s.gcr.io`. If the
+{{< note >}}
+For Kubernetes v1.23.0 through to v1.23.14, kubeadm used images from `k8s.gcr.io` by default. See [registry.k8s.io: faster, cheaper and Generally Available](/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/).
```

Comment:
D'oh. This should be a link to https://k8s.io/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/ and _not_ the shortened version we usually ask for.

Reply:
#38468 to resolve

Reply:
Why do we need the full URL? @sftim

Reply:
Since the blog was released during 1.25, the shorthand for the article link doesn't resolve to anything on the v1-24.docs, v1-23.docs, or v1-22.docs subdomains and the reader gets a 404. By explicitly setting the url to k8s.io we can ensure the reader is redirected to the latest version of docs which include the article. 

You can test it out on live by going to: 
https://v1-24.docs.kubernetes.io/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/
https://v1-23.docs.kubernetes.io/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/
https://v1-22.docs.kubernetes.io/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/

vs 
https://v1-25.docs.kubernetes.io/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/